plumber/api/importer.py

```diff
@@ -49,68 +45,171 @@ class ParallelImportBuilder:
     def __init__(
         self,
         file_system: GameFileSystem,
+        # General settings
+        import_materials: bool = True,
+        import_lights: bool = True,
+        light_factor: float = 1.0,
+        sun_factor: float = 1.0,
+        ambient_factor: float = 1.0,
+        import_sky_camera: bool = True,
+        sky_equi_height: int = 1024,
+        scale: float = 1.0,
+        target_fps: float = 30.0,
+        remove_animations: bool = False,
+        import_unknown_entities: bool = False,
         # Material options (applied to all imports in this builder)
         simple_materials: bool = False,
         texture_format: str = "Png",
         texture_interpolation: str = "Linear",
         allow_culling: bool = False,
         editor_materials: bool = False,
+        # VMF-specific settings
+        vmf_import_brushes: bool = True,
+        vmf_import_overlays: bool = True,
+        vmf_epsilon: float = 0.01,
+        vmf_cut_threshold: float = 0.1,
+        vmf_merge_solids: str = "MERGE",
+        vmf_invisible_solids: str = "SKIP",
+        vmf_import_props: bool = True,
+        vmf_import_entities: bool = True,
+        vmf_import_sky: bool = True,
+        vmf_scale: float = 1.0,
+        # MDL-specific settings
+        mdl_import_animations: bool = True,
+        # Collection options (applied to all imports)
+        main_collection=None,
+        brush_collection=None,
+        overlay_collection=None,
+        prop_collection=None,
+        light_collection=None,
+        entity_collection=None,
+        apply_armatures: bool = False,
```

Comment:
Apply armatures is not a collection setting (only passed to same place as an implementation detail). Should be under MDL settings in the API surface